darwin/client.py

```diff
@@ -360,7 +360,7 @@ def create_dataset(self, name: str, team: Optional[str] = None) -> RemoteDataset
             team=team or self.default_team,
             slug=dataset["slug"],
             dataset_id=dataset["id"],
-            item_count=dataset["num_images"] + dataset["num_videos"],
+            item_count=dataset["num_images"],
```

Comment:
having "num_videos" made sense to me, what was the issue this was causing?

Reply:
There's no `num_videos` in the `show.json` render view in backend. `create_dataset` fails if I include `num_videos`

Reply:
oki!